discord/ext/commands/cog.py

```diff
@@ -183,6 +184,7 @@ def __new__(cls, *args: Any, **kwargs: Any) -> Self:
         attrs['__cog_group_name__'] = group_name
         attrs['__cog_group_nsfw__'] = kwargs.pop('group_nsfw', False)
         attrs['__cog_group_auto_locale_strings__'] = kwargs.pop('group_auto_locale_strings', True)
+        attrs['__cog_group_extras__'] = attrs.get('extras', kwargs.pop('extras', {}))
```

Comment:
It needs to be called `group_extras`.

This also needs documentation.

Reply:
Added in recent commits.